$MirOS: ports/emulators/qemu/patches/patch-dis-asm_h,v 1.2 2005/05/03 19:30:03 tg Exp $
--- dis-asm.h.orig	Wed Apr 27 20:51:43 2005
+++ dis-asm.h	Tue May  3 18:02:23 2005
@@ -13,6 +13,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <inttypes.h>
+#include "config.h"
 
 #define PARAMS(x) x
 typedef void *PTR;
